website_event_snippet_calendar/controllers/main.py

```diff
@@ -48,7 +48,7 @@ def days_with_events(self, start, end):
         type="json",
         website=True,
     )
-    def events_for_day(self, day=None, limit=None):
+    def events_for_day(self, day=None, limit=None, enddateFlag=None):
```

Comment:
OCA's convention is to name variables with snake case (https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#342variable-names)

Reply:
Correct it. Thanks!

Reply:
Improved it. Thanks!